app/konnect/gateway-manager/plugins/add-custom-plugin.md

```diff
@@ -0,0 +1,172 @@
+for more information.
+
+{:.important}
+> **Caution**: Carefully test the operation of any custom plugins before deploying
+them to production. Kong is not responsible for the operation or support of any 
+custom plugins, including any performance impacts on your {{site.konnect_short_name}}
+or {{site.base_gateway}} deployments. 
+
+## Prerequisites
+
+You have a custom {{site.base_gateway}} plugin with a `schema.lua` file.
+The custom plugin can be written in any supported language, but the schema file must be in Lua.
+For help with developing plugins, see the [plugin development resources](#more-information).
+  
+If you have a custom plugin written in a language other than Lua, convert the schema 
+into a `schema.lua` file before uploading it to {{site.konnect_short_name}}.
+
+If using the Custom Plugins API, you have a personal access token or a system account
+token to authenticate with the API. You can pass your token with any API request using an 
+authentication header:
+
+```
+--header 'Authorization: Bearer kpat_xgfT'
+```
+
+## Add a custom plugin to a {{site.konnect_short_name}} org
+
+{{site.konnect_short_name}} only requires the custom plugin's `schema.lua` file. 
+Using that file, it creates a plugin entry in the plugin catalog for your control plane.
+
+Upload a custom plugin schema to create a configurable entity in {{site.konnect_short_name}}:
+
+{% navtabs %}
+{% navtab Konnect UI %}
+
+1. From the **Gateway Manager**, open a control plane.
+1. Open **Plugins** from the side navigation, then click **Add Plugin**.
+1. Open the **Custom Plugins** tab, then click **Create** on the Custom Plugin tile.
+1. Upload the `schema.lua` file for your plugin.
+1. Check that your file displays correctly in the preview, then click **Save**.
+
+{% endnavtab %}
+{% navtab Konnect API %}
+
+Upload the `schema.lua` file for your plugin using the `/plugin-schemas` endpoint:
+
+```sh
+curl -i -X POST \
+  https://{region}.api.konghq.com/v2/{controlPlaneId}/core-entities/plugin-schemas \
+  --data lua_schema=@example-schema.lua
+```
+
+This example specifies a file, but you can also include the entire schema in the request as JSON.
+
+You should get an HTTP 201 response. You can check that your schema was uploaded using the following request:
+
+```sh
+curl -i -X GET \
+  https://{region}.api.konghq.com/v2/{controlPlaneId}/core-entities/plugin-schemas
+```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+Uploading a custom plugin schema adds the plugin to a specific control plane. 
+If you need it to be available in multiple control planes, add the schema individually to each one.
+
+You can now configure this custom plugin like any other plugin in {{site.konnect_short_name}}.
+
+## Upload files to data plane nodes
+
+After uploading a schema to {{site.konnect_short_name}}, 
+upload all plugin files to **each** {{site.base_gateway}} data plane node:
+* Basic modules: `schema.lua`, `handler.lua`
+* Advanced modules: `api.lua`, `daos.lua`, `migrations.lua`
```

Comment:
We can remove this reference to Advanced Modules as Konnect does not support these modules (ref: https://kongstrong.slack.com/archives/C03LRB400TC/p1694035306780859)